src/interactions/interaction.ts

```diff
@@ -11,8 +11,53 @@ module Plottable {
      */
     protected _componentToListenTo: Component;
 
-    public _anchor(component: Component) {
+    private _anchorCallback = (component: Component) => this._anchor(component);
+
+    private _isAnchored: Boolean;
+
+    /* tslint:disable: */
```

Comment:
@jtlan Intentional leftover, so I don't forget. Basically I need a no-op function (kind of abstract) in the base class. But the tslint does not allow me to do that. So I needed to invent an instance variable which is never ever used. Do you think it is alright? Also tslint disable does not work